CMakeLists.txt

```diff
@@ -204,7 +220,11 @@ IF(BOOST_STATIC)
     set(Boost_USE_STATIC_LIBS   ON)
 endif()
 
-find_package(OpenSceneGraph 3.2.0 REQUIRED osgDB osgViewer osgText osgGA osgAnimation osgParticle osgQt osgUtil osgFX)
+if (NOT ANDROID)
+    find_package(OpenSceneGraph 3.2.0 REQUIRED osgDB osgViewer osgText osgGA osgAnimation osgParticle osgQt osgUtil osgFX)
```

Comment:
For the QT stuff, it might be better to add a new variable USE_QT that controls use of osgQt, the finding of QT libraries and the compilation of QT code in components/. USE_QT would be set to false if BUILD_LAUNCHER and BUILD_OPENCS are both disabled.